CHEWBBACA/CHEWBBACA_NS/stats_requests.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+AUTHORS
+
+    Mickael Silva
+    github: @
```

Comment:
We're going to remove that part with the authors, the module will start with information about usage and contacts will be shown through other method that includes all authors and the group email.

Reply:
This will be shown at the start of every process:
```
chewBBACA version: 2.5.0
Authors: Mickael Silva, Pedro Cerqueira, Rafael Mamede
Github: https://github.com/B-UMMI/chewBBACA
Wiki: https://github.com/B-UMMI/chewBBACA/wiki
Tutorial: https://github.com/B-UMMI/chewBBACA_tutorial
Contacts: imm-bioinfo@medicina.ulisboa.pt
```